gossip/simulation/gossip.go

```diff
@@ -15,6 +15,8 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
+// +build ignore
```

Comment:
Oh, because it's `package main` and you want to use `go run` instead of giving it its own directory where it can use `go build`? That seems reasonable but I'd add a comment.